db/predicate.go

```diff
@@ -10,17 +10,12 @@ import (
 )
 
 type Predicate struct {
-	Field              string
-	Operator           OperatorType
-	Value              any
-	RelationFieldNames []string
-	And                []*Predicate
-	Or                 []*Predicate
-}
-
-type CountOption struct {
-	Column string
-	Unique bool
```

Comment:
why delete these 2 types

Reply:
@khaicm-0897 the `CountOption` is now merged in to the `QueryOption`